clang/test/SemaSYCL/args-size-overflow.cpp

```diff
@@ -1,36 +1,25 @@
-// RUN: %clang_cc1 -fsycl -triple spir64 -fsycl-is-device -fsyntax-only -verify %s
-// RUN: %clang_cc1 -fsycl -triple spir64 -Werror=sycl-strict -DERROR -fsycl-is-device -fsyntax-only -verify %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -internal-isystem %S/Inputs -fsyntax-only -Wsycl-strict -sycl-std=2020 -verify %s
```

Comment:
Added `-sycl-std=2020` flag so we don't get this warning : `Passing of kernel functions by reference is a SYCL 2020 extension` , since the original test is about warning related to size of kernel arguments.

Reply:
Also, removed this test from `sycl/test/basic_tests/args-size-overflow.cpp`